### ISSUE TITLE
Confusion about Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space

### ISSUE THREAD

**Issue by ABChh26** [Degiacomi-Lab/molearn#3]
Hi, there! I was reading about Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space and I saw that simulated conformations of open and closed states are used as input, is the data from both states mixed and used as input to the model? Or is one state of protein input to the model alone for training? If it is a mixture of both states, wouldn't the generated results combine the features of both states of the conformation?

**Comment by SCMusson** [comment]
The dataset fed to the neural network features examples from both the open and closed state, without any artificial intermediates. The architecture we use is an autoencoder [(see here)](https://en.wikipedia.org/wiki/Autoencoder) which we use to extract collective variables / generate a low dimensional representation of the data. We then can use the autoencoder to generate structures corresponding to interpolations between the two states along those collective variables in hope that we can capture transition structures. The autoencoder should be capable of reproducing the input structures pretty closely and interpolations between those structures would, as you put, 'combine the features of both states'. However, this is exactly what we would be looking for in structures along the transition pathway. A transition structure is halfway between two states and we would therefore expect it to combine features of both states.
I will add that this repository is primarily concerned with [our latest paper](https://journals.aps.org/prx/abstract/10.1103/PhysRevX.11.011052) where we build on the paper you're reading by minimizing bond, angle, torsion, and nonbonded energies so that the network learns to combine the features of both states in a way that is more reasonable from the perspective of molecular physics.

**Comment by ABChh26** [comment]
Thanks a lot for the answer! Well, let me explain my understanding that your subject undergoes a conformational transitions&nbsp;in the simulation, so you write the conformational datas from the beginning to the endless of this&nbsp;conformational transitions&nbsp;into a ".pdb" file and then feed it into the neural network.I noticed the molearn project before you prompted me, adding the idea of convolution to process the data and build a loss function calculation based on the force field.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "degiacom/molearn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月7日(星期二) 下午5:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [degiacom/molearn] Confusion about Coupling Molecular Dynamics and Deep Learning to Mine Protein Conformational Space (Issue #3)





 
The dataset fed to the neural network features examples from both the open and closed state, without any artificial intermediates. The architecture we use is an autoencoder (see here) which we use to extract collective variables / generate a low dimensional representation of the data. We then can use the autoencoder to generate structures corresponding to interpolations between the two states along those collective variables in hope that we can capture transition structures. The autoencoder should be capable of reproducing the input structures pretty closely and interpolations between those structures would, as you put, 'combine the features of both states'. However, this is exactly what we would be looking for in structures along the transition pathway. A transition structure is halfway between two states and we would therefore expect it to combine features of both states.
 I will add that this repository is primarily concerned with our latest paper where we build on the paper you're reading by minimizing bond, angle, torsion, and nonbonded energies so that the network learns to combine the features of both states in a way that is more reasonable from the perspective of molecular physics.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
